Userland/Libraries/LibCore/FileWatcher.h

```diff
@@ -13,48 +13,114 @@
 #include <AK/RefCounted.h>
 #include <AK/Result.h>
 #include <AK/String.h>
+#include <Kernel/API/InodeWatcherEvent.h>
+#include <Kernel/API/InodeWatcherFlags.h>
 #include <LibCore/Notifier.h>
 
 namespace Core {
 
 struct FileWatcherEvent {
     enum class Type {
-        Modified,
-        ChildAdded,
-        ChildRemoved,
+        MetadataModified,
+        ContentModified,
```

Comment:
Since we always update the `mtime` after writing to an inode, is the separate `ContentModified` actually useful? You will always get a `MetadataModified` anyway.. I think(?)